client/lib/plans/constants.js

```diff
@@ -105,6 +148,41 @@ export const featuresList = {
 		getTitle: () => i18n.translate( 'Monetize Your Site' ),
 		getDescription: () => i18n.translate( 'Add advertising to your site through our WordAds program and get paid.' ),
 		plans: allPaidPlans
+	},
+
+	[ FEATURE_FREE_SITE ]: {
+		getTitle: () => i18n.translate( 'Free site' ),
+		plans: allPlans
+	},
+
+	[ FEATURE_WP_SUBDOMAIN ]: {
+		getTitle: () => i18n.translate( 'WordPress.com subdomain' ),
+		plans: [ PLAN_FREE ]
```

Comment:
Hmm this could get out of sync too. If we need the reverse lookup maybe lets remove this, and add in helper function like:

```javascript
getPlansForFeature( feature )
```

We could also memoize the results if this is called frequently